dataikuapi/dss/admin.py

```diff
@@ -798,6 +798,41 @@ def set_definition(self, definition):
             "PUT", "/admin/globalAPIKeys/%s" % self.key,
             body = definition)
 
+class DSSPersonalApiKey(object):
+    """
+    A personal API key on the DSS instance
+    """
+    def __init__(self, client, key):
+        self.client = client
+        self.key = key
+
+    ########################################################
+    # Key deletion
+    ########################################################
+
+    def delete(self):
+        """
+        Delete the api key
+
+        Note: this call requires an API key with admin rights
```

Comment:
Should be removed given users can delete their own keys.

Reply:
yes, definitively: it was a copy-paste artefacts